### PR TITLE
Rhosp10

### PR DESCRIPTION
**Pull request by redhat-rmcallis**
#### Description:

code to run checks against rhosp10.

#### Rationale:

This code was meant to backport a number of checks used in rhosp13 (non containerized checks) to rhosp10 in support of a RH consulting engagement. This will also fill in the gaps for  other customers wanting to scan their rhosp 10 installations against cui or STIG.

### CODE REVIEW

**Comment by yuumasato** [comment]
@openscap-ci ok to test

**Comment by yuumasato** [comment]
@openscap-ci add to whitelist

File: CMakeLists.txt [hunk 1]
```diff
@@ -243,6 +244,7 @@ message(STATUS "openSUSE: ${SSG_PRODUCT_OPENSUSE}")
 message(STATUS "RHEL 6: ${SSG_PRODUCT_RHEL6}")
 message(STATUS "RHEL 7: ${SSG_PRODUCT_RHEL7}")
 message(STATUS "RHEL 8: ${SSG_PRODUCT_RHEL8}")
+message(STATUS "RHOSP10: ${SSG_PRODUCT_RHOSP13}")
```
Comment (jan-cerny) [inline CMakeLists.txt]: This should be RHOSP10

File: applications/openstack/cinder/cinder_conf_file_perms/bash/shared.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# platform = Red Hat OpenStack Platform 10
+for file in /etc/cinder/cinder.conf \
+		/etc/cinder/rootwrap.conf; do
+	chown root $file
+	chgrp cinder $file
```
Comment (jan-cerny) [inline applications/openstack/cinder/cinder_conf_file_perms/bash/shared.sh]: add quotes

File: applications/openstack/cinder/cinder_conf_file_perms/rule.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+prodtype: rhosp10,rhosp13
+
+title: 'Check-Block-02: Are strict permissions set for cinder config files?'
+title: 'Verify Permissions on SSH Server Public *.conf Key Files'
```
Comment (jan-cerny) [inline applications/openstack/cinder/cinder_conf_file_perms/rule.yml]: The rule should have only 1 title.

File: applications/openstack/cinder/cinder_conf_file_perms/rule.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+description: '{{{ describe_file_permissions(file="/etc/cinder/*.conf", perms="0640") }}}'
+
+rationale: |-
+    Due to the nature of the cinder config files, normal users should no be
```
Comment (jan-cerny) [inline applications/openstack/cinder/cinder_conf_file_perms/rule.yml]: should not be

File: rhosp10/profiles/cui.profile [hunk 5]
```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP CUI'
+
+description: 'Sample profile description.'
```
Comment (jan-cerny) [inline rhosp10/profiles/cui.profile]: Please add a better description.

File: rhosp10/overlays/nist_support.xml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+</Rule>
+
+<Rule id="nist_met_inherently">
+<title>Product Meets this Requirement</title>
+<rationale>
+Red Hat Enterprise Linux meets this requirement through design and implementation.
+</rationale>
+<ocil>RHEL6 supports this requirement and cannot be configured to be out of
```
Comment (jan-cerny) [inline rhosp10/overlays/nist_support.xml]: RHEL6?

File: applications/openstack/cinder/cinder_conf_file_perms/bash/shared.sh (hunk 2 above)
Comment (redhat-rmcallis) [reply applications/openstack/cinder/cinder_conf_file_perms/bash/shared.sh]: quotes added around variable

File: rhosp10/overlays/nist_support.xml (hunk 6 above)
Comment (redhat-rmcallis) [reply rhosp10/overlays/nist_support.xml]: removed rhel6 and just made general reference to RHEL.

File: rhosp10/profiles/cui.profile (hunk 5 above)
Comment (redhat-rmcallis) [reply rhosp10/profiles/cui.profile]: description added.

File: rhosp10/overlays/nist_support.xml (hunk 6 above)
Comment (redhat-rmcallis) [reply rhosp10/overlays/nist_support.xml]: updated.

File: applications/openstack/cinder/cinder_conf_file_perms/rule.yml [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    Due to the nature of the cinder config files, normal users should not be
+    able to view contents
+severity: medium
+
+references:
+    cui: 3.1.5
+
+ocil_clause: '{{{ ocil_clause_file_permissions(file="/etc/cinder/*.conf", perms="-rw-r--r--") }}}'
+
+ocil: '{{{ ocil_file_permissions(file="/etc/cinder/*.conf", perms="-rw-r--r--") }}}'
```
Comment (jan-cerny) [inline applications/openstack/cinder/cinder_conf_file_perms/rule.yml]: This is not 0640, but 0644. Chnage this to "-rw-r-----"

File: rhosp10/profiles/stig.profile [hunk 8]
```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP STIG'
+
+description: 'Sample profile description.'
```
Comment (jan-cerny) [inline rhosp10/profiles/stig.profile]: It's still missing a proper description.

File: rhosp10/profiles/stig.profile [hunk 9]
```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP STIG'
```
Comment (redhatrises) [inline rhosp10/profiles/stig.profile]: @redhat-rmcallis please change this to `[DRAFT] STIG for Red Hat OpenStack Plaform 10`

File: rhosp10/profiles/cui.profile [hunk 10]
```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP CUI'
```
Comment (redhatrises) [inline rhosp10/profiles/cui.profile]: @redhat-rmcallis please change this to `[DRAFT] Controlled Unclassified Infomration (CUI) Profile for Red Hat OpenStack Plaform 10`

Comment (redhat-rmcallis) [reply rhosp10/profiles/cui.profile]: changed

**Pull request merged**